internal/gatewayapi/securitypolicy.go

```diff
@@ -892,6 +889,31 @@ func (t *Translator) buildExtAuth(policy *egv1a1.SecurityPolicy, resources *Reso
 	return extAuth, nil
 }
 
+func backendRefAuthority(resources *Resources, backendRef *gwapiv1.BackendObjectReference, policy *egv1a1.SecurityPolicy) string {
+	if backendRef == nil {
+		return ""
+	}
+
+	backendNamespace := NamespaceDerefOr(backendRef.Namespace, policy.Namespace)
+	backendKind := KindDerefOr(backendRef.Kind, KindService)
+	if backendKind == egv1a1.KindBackend {
+		backend := resources.GetBackend(backendNamespace, string(backendRef.Name))
+		if backend != nil {
+			// TODO: exists multi FQDN endpoints?
```

Comment:
I think that we need to support user-defined authority for cases where there are multiple endpoints. We can leave this to a separate PR.

Reply:
yes, telemetry also need it.